### PR TITLE
feat: add support for additional claims

### PR DESCRIPTION
**Pull request by JustinBeckwith**
This is required to land https://github.com/google/google-auth-library-nodejs/issues/134

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -42,11 +43,12 @@ export class GoogleToken {
   key: string|undefined;
   keyFile: string|undefined;
   iss: string|undefined;
-  sub: string;
+  sub: string|undefined;
```
Comment (jinwoo) [inline src/index.ts]: `sub?: string;` ?

File: src/index.ts [hunk 2]
```diff
@@ -42,11 +43,12 @@ export class GoogleToken {
...
   scope: string|undefined;
   rawToken: string|null;
   tokenExpires: number|null;
   email: string;
+  additionalClaims?: {};
```
Comment (jinwoo) [inline src/index.ts]: Why are all these fields public?

File: src/index.ts [hunk 3]
```diff
@@ -168,7 +170,8 @@ export class GoogleToken {
         sub: this.sub,
         key: this.key,
         keyFile: this.keyFile,
-        scope: this.scope
+        scope: this.scope,
+        additionalClaims: this.additionalClaims
```
Comment (jinwoo) [inline src/index.ts]: Nit. If you put a comma at the end of the last item, you wouldn't have to change that line when you append another item later.

Comment (JustinBeckwith) [reply src/index.ts]: That sounds like the kind of thing we should add to the linter if we want to enforce it :)

File: src/index.ts (hunk 2 above)
Comment (JustinBeckwith) [reply src/index.ts]: It's kinda weird, but it's just how the library was originally written ::shrugs::

File: src/index.ts (hunk 3 above)
Comment (JustinBeckwith) [reply src/index.ts]: Having been said, fixed

**Pull request merged**